vspace/src/main/java/edu/asu/diging/vspace/web/staff/AddSequenceController.java

```diff
@@ -35,9 +38,16 @@ public String showAddSequence(@PathVariable("id") String moduleId, Model model)
     @RequestMapping(value = "/staff/module/{moduleId}/sequence/add", method = RequestMethod.POST)
     public String addSequence(Model model, @PathVariable("moduleId") String moduleId, @ModelAttribute SequenceForm sequenceForm,
             Principal principal) {
-
+    	
         sequenceManager.storeSequence(moduleId, sequenceForm); 
-
+        // Added by Prashant Jadhav for story id 22 start
+        LinkedList<ISequence> ll= new LinkedList<>(moduleManager.getModuleSequences(moduleId));
+        if(ll.size()==1) {
+        	IModule module=moduleManager.getModule(moduleId);
+        	module.setStartSequence(ll.get(0));
+        	moduleManager.updateModule(module);
+        }
+        // Added by Prashant Jadhav for story id 22 end
```

Comment:
AddSequence just adds a sequence. Setting the start sequence is a different feature. There should be a user interface that let's a user select a start sequence. Also, why do you use a linked list instead of the one that `getModuleSequence` returns?